### ISSUE TITLE
AMQP + TLS Integration disabled ?

### ISSUE THREAD

**Issue by jbnoblot** [chirpstack/chirpstack#502]
Hello,
Today I update my chirpstack but my integration with my RabbitMQ server in amqps not working anymore.

I am not practice rustlang but after search this commit [https://github.com/chirpstack/chirpstack/commit/66ab41036b3afedc76049b6e56b98ae65aa3b37e](https://github.com/chirpstack/chirpstack/commit/66ab41036b3afedc76049b6e56b98ae65aa3b37e) seems break tls cause disabled rustls.

My english is poor but If I understand you waiting features from lapin-rs cause. 
It's this flag `rustls--ring` from this commit [https://github.com/amqp-rs/lapin/commit/26f956b76251a08c26d257f5f2efabfa7dcba4ca](https://github.com/amqp-rs/lapin/commit/26f956b76251a08c26d257f5f2efabfa7dcba4ca) fix the issues you have ?

**Comment by brocaar** [comment]
Yes, there is an issue with the latest `lapin` crate that breaks the `rustls` build as it enables the default `rustls` features, which overrules the backend selection (ChirpStack is using the `ring` backend). We need to wait for a `lapin` crate update until this is fixed. This is similar to: https://github.com/amqp-rs/lapin/issues/414
